### PR TITLE
Allow setting session cookie domain via configuration.

### PR DESCRIPTION
**Pull request by mhennoch**

### CODE REVIEW

**Review by t2t2 — COMMENTED** [review]
Document option somewhere too

File: README.md [hunk 1]
```diff
@@ -85,6 +85,7 @@ can be used within either Web or Service Workers for manual instrumentation.
 | app | string | Application name | 'unknown-browser-app' |
 | environment | string | Sets a value for the `environment` attribute (persists through calls to `setGlobalAttributes()`) | (No default) |
 | globalAttributes | object | Extra attributes to add to each reported span.  See also `setGlobalAttributes` | {} |
+| cookieDomain | string | Sets session cookie to this domain | If unspecified, it defaults to the same host that set the cookie, excluding subdomains
```
Comment (jtmalinowski) [inline README.md:88]: Please add a CHANGELOG entry

File: integration-tests/tests/cookies/cookies.spec.js [hunk 2]
```diff
@@ -58,5 +58,16 @@ module.exports = {
...
     await browser.end();
-  }
+  },
+  'setting invalid cookieDomain in options does not set the cookie': async function(browser) {
+    await browser.url(browser.globals.getUrl('/cookies/cookies-domain.ejs'));
+
+    await browser.globals.findSpan(span => span.name === 'documentFetch');
+    const cookie = await browser.getCookie('_splunk_rum_sid');
+
+    await browser.assert.not.ok(cookie);
```
Comment (jtmalinowski) [inline integration-tests/tests/cookies/cookies.spec.js:68]: Should we have a test which checks if setting custom domain actually works?

**Review by jtmalinowski — CHANGES_REQUESTED** [review]

**Review by johnbley — APPROVED** [review]

File: integration-tests/tests/cookies/cookies.spec.js (hunk 2 above)
Comment (mhennoch) [reply integration-tests/tests/cookies/cookies.spec.js:68]: Will try to do it

File: README.md (hunk 1 above)
Comment (mhennoch) [reply README.md:88]: Done

**Review by jtmalinowski — APPROVED** [review]

**Review by t2t2 — APPROVED** [review]

**Pull request merged**